### ISSUE TITLE
✨ Admin UI for new Workstream model

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#12081]
## ✨ Feature

We updated the workstream model in #7574. Now we need a way for admins to manage them.

## 🕵️ Details

Admin pages (using our new patterns) for workstream including:
- Work stream index table (URL: **/admin/settings/work-streams**)
- Create work stream page
- Work stream information page (static)
- Edit work stream information

## 🎨 Design
[Link to figma file](https://www.figma.com/design/Vel9dQKKSxTvokArhYbFQY/Admin---Work-streams?node-id=16032-12628&t=j6qoati33eCUL5Kp-1) by @Jerryescandon 

![image](https://github.com/user-attachments/assets/620cddf1-b2c9-4175-bdbe-547d1d8850ac)

## ✅ Acceptance Criteria

- [ ] Work stream index table page (URL: /admin/settings/work-streams)
  - [ ] Default sort should be grouped by Community id, and then sorted alphabetically by localized name (english or french)
  - [ ] If sorting by Community is difficult at all, just by name is fine
  - [ ] Searching is out of scope
  - [ ] default page size of 50
- [ ] Create work stream page
- [ ] View Work stream page (Advanced Tools tab out of scope)
- [ ] Edit work stream information page, 

## 🛑 Blockers

```[tasklist]
### Blocked By
- [ ] #7574
```
